<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QWidget" name="LoginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="header_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>601</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Narrow</family>
     <pointsize>32</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel#header_label {
	font-family: Arial Narrow;
	color: black;
	font-weight: bold;
}</string>
   </property>
   <property name="text">
    <string>Bulk Club Login</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="username_label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>100</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel#username_label {
	font-size: 20px;
	color: white;
}</string>
   </property>
   <property name="text">
    <string>Username:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="password_textbox">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>140</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit#password_textbox {
	color: white;
	background: black;
	font-family: arial;
	font-siz	e: 16px;
	border:1px solid #ffffff;
	border-radius: 2px;
}

QLineEdit#password_textbox:hover {
	background: #191b1c;
	border: 2px solid white;
}</string>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="password_label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>140</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel#password_label {
	color: white;
	font-size: 20px;
}</string>
   </property>
   <property name="text">
    <string>Password:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="user_type_label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>190</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel#user_type_label {
	font-size: 18px;
	color: white;
}</string>
   </property>
   <property name="text">
    <string>Type of User:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="login_button">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>250</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#login_button{
	background: gray;
	color: black;
	font-size: 16px;
	border:2px solid #000000;
	border-radius: 5px;
}

QPushButton#login_button:hover{
	background: #b2b9bf;
}</string>
   </property>
   <property name="text">
    <string>Login Now</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="username_textbox">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>100</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit#username_textbox {
	color: white;
	background: black;
	font-family: arial;
	font-siz	e: 16px;
	border:1px solid #ffffff;
	border-radius: 2px;
}

QLineEdit#username_textbox:hover {
	background: #191b1c;
	border: 2px solid white;
}</string>
   </property>
  </widget>
  <widget class="QComboBox" name="user_type_combo">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>190</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox#user_type_combo{
	color: white;
	font-family: arial;
	background: pale;
	border:1px solid #ffffff;
	border-radius: 2px;
}

QComboBox#user_type_combo:hover {
	background: #191b1c;
	border: 2px solid white;
}</string>
   </property>
   <item>
    <property name="text">
     <string>Member</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Administrator</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Store Manager</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
